docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -66,27 +54,28 @@ spec:
 And create it:
 
 ```shell
-$ kubectl create -f resourcedefinition.yaml
-customresourcedefinitions "crontabs.stable.example.com" created
+kubectl create -f resourcedefinition.yaml
 ```
 
 Then a new RESTful API endpoint is created at:
 
-`/apis/stable.example.com/v1/namespaces/<namespace>/crontabs/...`
+```
+/apis/stable.example.com/v1/namespaces/<namespace>/crontabs/...
```

Comment:
maybe clarify that `<namespace>` comes from the current context? (if that is the case)